### PR TITLE
remove bpf_snprintf

### PR DESCRIPTION
**Pull request by bfforever**
**What type of PR is this?**
/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: bpf/include/bpf_log.h [hunk 1]
```diff
@@ -30,11 +36,29 @@ struct log_event {
     __u32 ret;
     char msg[MAX_MSG_LEN];
 };
+
+// 
+struct __attribute__((packed)) log_event_v0 {
+    __u32 ret;
+    __u32 flag;
```
Comment (hzxuzhonghu) [inline bpf/include/bpf_log.h:43]: what does ret and flag mean

File: bpf/include/bpf_log.h [hunk 2]
```diff
@@ -116,5 +140,29 @@ static inline int map_lookup_log_level()
...
+            e = bpf_ringbuf_reserve(&kmesh_events_v0, sizeof(struct log_event_v0), 0);                                 \
+            if (!e)                                                                                                    \
+                break;                                                                                                 \
+            e->fmt = fmt;                                                                                              \
+            e->arg0 = arg0;                                                                                            \
+            e->arg2 = arg1;                                                                                            \
+            flag |= ARG_0;                                                                                             \
+            flag |= ARG_2;                                                                                             \
+            e->flag = flag;                                                                                            \
```
Comment (hzxuzhonghu) [inline bpf/include/bpf_log.h:163]: NVM, i understand what flag means

File: pkg/logger/logger.go [hunk 3]
```diff
@@ -171,6 +203,32 @@ func handleLogEvents(ctx context.Context, rbMap *ebpf.Map) {
 	}
 }
 
+func handleLogEventsV0(ctx context.Context, rbMapV0 *ebpf.Map) {
+	log := NewLoggerScope("ebpf")
+	events, err := ringbuf.NewReader(rbMapV0)
+	if err != nil {
+		log.Errorf("ringbuf new reader from rb map v0 failed:%v", err)
```
Comment (LiZhenCheng9527) [inline pkg/logger/logger.go:210]: What's v0 means?

**Pull request closed without merging**